docs/manual/troubleshooting.rst

```diff
@@ -71,7 +71,72 @@ Qtile, so that the debug logs printed by Qtile are only the server's.
 
 If you suspect a client may be responsible for a bug, it can be helpful to look
 at the issue trackers for other compositors, such as `sway
-<https://github.com/swaywm/sway/issues`_. Similarly if you're hacking on
+<https://github.com/swaywm/sway/issues>`_. Similarly if you're hacking on
```

Comment:
Ha. I think there are 3 open PRs with this fix in...

Reply:
:grinning: